packages/vscode/src/telemetry.ts

```diff
@@ -1,7 +1,7 @@
 import TelemetryReporter from 'vscode-extension-telemetry'
 
 export class Telemetry {
-  private readonly key: string = ''
+  private readonly key: string = 'fe968a4a-7cbf-4c43-bf6e-3e968f3c4519'
```

Comment:
We need a way to differentiate between the Insider and the stable extension - with this I assume everything would be reported to this ID.

(Fine to do in a future PR and merge this without - but needs to be done before 2.3.0)